learning-guide/beginner.md

```diff
@@ -15,6 +18,8 @@ Click the image to view on YouTube.
 | --- | :-- | --- |
 | The Complete Guide to Full Stack Ethereum Development - Tutorial for Beginners [2021] | [![video](https://img.youtube.com/vi/a0osIaAOFSE/0.jpg)](https://www.youtube.com/watch?v=a0osIaAOFSE) | 1h 01m 04s
 | Local Solidity Development | [View](https://hardhat.org/) |
+| Solidity, Blockchain, and Smart Contract Course – Beginner to Expert Python Tutorial | [![video](https://user-images.githubusercontent.com/68073132/135998915-31e05399-0495-493a-920a-1a3ddd939cf2.jpeg)](https://youtu.be/M576WGiDBdQ) | 16h 22m 12s
+| How to Build a Full Stack NFT Marketplace on Ethereum with Polygon and Next.js | [![video](https://user-images.githubusercontent.com/68073132/136000852-6c23261c-d1f1-4c0f-ba53-c406c6b6ce14.jpeg)](https://youtu.be/GKJBEEXUha0) | 2h 21m 22s
```

Comment:
> It is best to separate the written content, just like the below:

## Development Guide

Click the image to view on YouTube.

| Title | Content | Length | 
| --- | :-- | --- |
| The Complete Guide to Full Stack Ethereum Development - Tutorial for Beginners [2021] | [![video](https://img.youtube.com/vi/a0osIaAOFSE/0.jpg)](https://www.youtube.com/watch?v=a0osIaAOFSE) | 1h 01m 04s
| Solidity, Blockchain, and Smart Contract Course – Beginner to Expert Python Tutorial | [![video](https://user-images.githubusercontent.com/68073132/135998915-31e05399-0495-493a-920a-1a3ddd939cf2.jpeg)](https://youtu.be/M576WGiDBdQ) | 16h 22m 12s
| How to Build a Full Stack NFT Marketplace on Ethereum with Polygon and Next.js | [![video](https://user-images.githubusercontent.com/68073132/136000852-6c23261c-d1f1-4c0f-ba53-c406c6b6ce14.jpeg)](https://youtu.be/GKJBEEXUha0) | 2h 21m 22s

## Written Content
| Title | Content |
| --- | :-- |
| Local Solidity Development | [View](https://hardhat.org/) |

Reply:
How about this one?

Reply:
Okay, I will change it right now.